src/components/table/__spec__.js

```diff
@@ -928,6 +928,18 @@ describe('Table', () => {
         expect(instance.tableContent.get(0)).toEqual(instance.props.children.get(0));
         expect(instance.tableContent.get(1)).toEqual(instance.emptyRow);
       });
+
+      describe('when a customEmptyRow has been provided', () => {
+        test('the custom empty row is rendered instead of the default', () => {
+          let data = Immutable.fromJS([]),
+              children = data.map((child, index) => { return <tr key={ index }></tr>; });
```

Comment:
If you aren't using a value in a function like this, but are forced to name it as you need the second, then using an `_` I think is best practice (in some languages at least):

`children = data.map((_, index) => { return <tr key={ index }></tr>; });`

Also, you can get rid of some of the clutter in this line with implied returns etc.

`children = data.map((_, index) => <tr key={ index }></tr>);`